.github/workflows/ci.yml

```
@@ -2,9 +2,9 @@ name: CI
 
 on:
     push:
-        branches: [main]
+        branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)